package.json

```diff
@@ -106,15 +110,24 @@
     "@types/react-redux": "7.1.0",
     "@types/redux-logger": "3.0.7",
     "@types/redux-mock-store": "1.0.0",
+    "@types/selenium-webdriver": "4.0.6",
+    "axe-core": "3.4.2",
+    "axe-sarif-converter": "2.3.0",
+    "axe-webdriverjs": "2.3.0",
+    "babel-jest": "23.6.0",
+    "chromedriver": "^84.0.1",
```

Comment:
looks like we're locking to specific versions

Reply:
Yeah, though I realised the Chrome browser on the pipeline gets updated and so the chrome driver equally has to update since similar versions work together.